routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
```

Comment:
For reference, what we've been doing is 
`if (!wordList.length) throw new Error('Word not found')`
But this line is not needed if you switch from `.find` to `.findOne`, which I think you should do.